Modules/TPC/run/tpcQCClusters_direct.json

```diff
@@ -28,31 +28,23 @@
         "className": "o2::quality_control_modules::tpc::Clusters",
         "moduleName": "QcTPC",
         "detectorName": "TPC",
-        "cycleDurationSeconds": "60",
+        "cycleDurationSeconds": "10",
```

Comment:
Are you sure about this one ? publishing every 10 seconds is quite a lot.

Reply:
Yes, this is for testing only anyway so it doesn't really matter.

Reply:
ok, thank you